packages/serverpod/test/database/columns/column_uuid_test.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Given a ColumnUuid', () {
+    var columnName = 'id';
+    var column = ColumnUuid(columnName);
+
+    test(
+        'when toString is called then column name withing double quotes is returned.',
+        () {
+      expect(column.toString(), '"$columnName"');
+    });
+
+    test('when columnName getter is called then column name is returned.', () {
+      expect(column.columnName, columnName);
+    });
+
+    test('when type is called then UuidValue is returned.', () {
+      expect(column.type, UuidValue);
+    });
+
+    group('with _ColumnDefaultOperations mixin', () {
+      test(
+          'when equals compared to NULL value then output is IS NULL expression.',
+          () {
+        var comparisonExpression = column.equals(null);
+
+        expect(comparisonExpression.toString(), '$column IS NULL');
+      });
+
+      test(
+          'when equals compared to uuid value then output is equals expression.',
+          () {
+        var comparisonExpression = column.equals(
+            UuidValue('testUuid', false /* Disable validation for test */));
```

Comment:
Suggestion Can also use one of the predefined static uuids to get a real valid value i.e: 
```suggestion
            UuidValue(Uuid.NAMESPACE_NIL);
```

The underlying value is `00000000-0000-0000-0000-000000000000`.

Reply:
Implemented your suggestion but preserved some of them so that I could match the value in the string.